### DISCUSSION TITLE
Get build time

### DISCUSSION THREAD

**Discussion by Copdate-Copdate** [fastlane/fastlane#17755] [General]
Hi,

I would like to pass, as a slack parameter, the total execution time but can't find any info about that.

Is fastlane provide such info, apart from the printed summary in the terminal?

**Comment by rogerluan** [comment]
Hey @Copdate-Copdate 👋 

Sounds like the option you're looking for is `analyze_build_time`. You can read more about it [in the docs](https://docs.fastlane.tools/actions/build_ios_app/).

Does that solve your issue?
